### DISCUSSION TITLE
How to enable logout menu?

### DISCUSSION THREAD

**Discussion by tht-activeviam** [atoti/atoti#531] [Q&A] [answered]
We enable the authentication in Atoti session and able to login.
But then we don't see the logout menu. There is no sub menu once you click the user menu, so there is no way to logout.
How do we enable it?

![image](https://user-images.githubusercontent.com/41277323/159662080-f4ee85e2-d515-451b-8c9b-5bbc41c638f4.png)

**Comment by tibdex** [comment, marked as answer]
Hi, logout support is currently being developed. You can follow the progress on #386. In the meantime, you can use private browsing or clearing the app cookies.
